presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -2,12 +2,11 @@
 LDAP Authentication
 ===================
 
-Presto can be configured to enable frontend LDAP authentication over
```

Comment:
Revert this change. The HTTPS is referring to the requirement on the Presto coordinator, not the backend connection to LDAP.